site/reference/pkg/init/README.md

```diff
@@ -9,25 +9,37 @@ description: >
     Initialize an empty package
 -->
 
-{{< asciinema key="pkg-init" rows="10" preload="1" >}}
+`init` initializes an existing empty directory as an empty kpt package.
 
-Init initializes an existing empty directory as an empty kpt package.
+### Synopsis
+<!--mdtogo:Long-->
+```
+kpt pkg init [DIR] [flags]
+```
+
+#### Args
+
+```
+DIR:
+  init fails if DIR does not already exist. Defaults to the current working directory.
```

Comment:
nit. verbatim for `init`?

Reply:
It is already inside a code block, so it is already in verbatim.